tgui/packages/tgui/interfaces/ScannerGate.js

```diff
@@ -103,6 +103,10 @@ const TARGET_SPECIES_LIST = [
     name: 'Synthetic Human',
     value: 'synthhuman',
   },
+  {
+    name: 'Teshari',
+    value: 'tesh',
```

Comment:
This wouldn't work otherwise, because that's not what your define is in the .dm file ;)
```suggestion
    value: 'teshari',
```